wiki/Tournaments/MWC/2020_4K/en.md

```diff
@@ -0,0 +1,223 @@
+---
+tags:
+- MWC 2020
+- MWC 4K 2020
+- MWC2020
+- MWC4K 2020
+---
+
+# osu!mania 4K World Cup 2020
+
+![MWC 4K 2020 Logo](/wiki/shared/news/banners/MWC4k2020.jpg)
+
+The **osu!mania 4K World Cup 2020** (***MWC 4K 2020***) is a country-based osu!mania tournament hosted by the [osu! team](/wiki/People/The_Team). It is the 7th installment of the osu!mania 4K World Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2020-07-16/2020-07-30 |
+| Live Drawings | 2020-08-08 14:00:00 UTC |
+| Group Stage | 2020-08-15/2020-08-16 |
+| Round of 16 | 2020-08-22/2020-08-23 |
+| Quarterfinals | 2020-08-29/2020-08-30 |
+| Semifinals | 2020-09-05/2020-09-06 |
+| Finals | 2020-09-12/2020-09-13 |
+| Grand Finals | 2020-09-19/2020-09-20 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | $150 per team member, profile badge, "osu!mania Champion" user title for one year |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | $80 per team member, profile badge |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | $40 per team member, profile badge |
+
+## Organisation
+
+The osu!mania 4K World Cup 2020 is run by the [osu! team](/wiki/People/The_Team) and various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
+| Map Selector | ![][flag_US] [-mint-](https://osu.ppy.sh/users/8976576), ![][flag_ID] [Mipha-](https://osu.ppy.sh/users/5767941), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
+| Commentator | *-- to be determined --* |
+| Referee | ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_CH] [Icerite](https://osu.ppy.sh/users/7226287), ![][flag_AU] [ill onion](https://osu.ppy.sh/users/8306102) ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Statistician | ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1106843)
+- [Livestream](https://www.twitch.tv/osulive)
+
+---
+
+## Ruleset
```

Comment:
delete newlines between list items

Reply:
After doing this it broke checks so no for now.